console.log("Handle error in async await");
